src/services/character/character.service.ts

```diff
@@ -1,31 +1,49 @@
-import _ from 'lodash'
-import { Document } from 'mongoose'
-import { ICharacter } from '../../types/character/character'
-import characters from '../../data/models/character-schema'
+import _ from "lodash";
+import mongoose from "mongoose";
+import { ICharacter } from "../../types/character/character";
 
 export class CharacterService {
-  private static _instance: CharacterService
+  private static _instance: CharacterService;
 
   public static getInstance(): CharacterService {
-    if(_.isNil(CharacterService._instance)) {
-      CharacterService._instance = new CharacterService()
+    if (_.isNil(CharacterService._instance)) {
+      CharacterService._instance = new CharacterService();
     }
-    return CharacterService._instance
+    return CharacterService._instance;
   }
 
-  public async getEntity(ownerId: string):Promise<Document | void> {
-    return await characters.findOne({ ownerId }, (err: Error, characterFound: Document): ICharacter => {        
-      if (characterFound) return characterFound.schema.obj
-    })
-    .catch((err): void => {
-      console.log(`Error: Character not found because: ${err}`)
-      return
-    })
+  public getEntity(_ownerId: string): Promise<ICharacter | never> {
+    return new Promise((resolve, reject): void => {
+      void mongoose
```

Comment:
I am curious about this one.
Adding void as prefix is usually required to avoid a lint error about unhandled promises.
You said earlier that the return type was not a promise so it should not be useless, right?

Reply:
The linter was still angry at it for some reason... Maybe it was still considering it a promise??

Reply:
See comment below.

Reply:
Ok.